config/crd/bases/awx.ansible.com_awxs.yaml

```diff
@@ -1535,6 +1535,150 @@ spec:
+                    type: array
+                  options:
+                    items:
+                      properties:
+                        name:
+                          type: string
+                        value:
+                          type: string
+                      type: object
+                    type: array
+                  searches:
+                    items:
+                      type: string
+                    type: array
+                type: object
+              redis_dns_policy:
+                type: string
+              redis_host_network:
+                type: boolean
+              ee_dns_config:
+                properties:
+                  nameservers:
+                    items:
+                      type: string
+                    type: array
+                  options:
+                    items:
+                      properties:
+                        name:
+                          type: string
+                        value:
+                          type: string
+                      type: object
+                    type: array
+                  searches:
+                    items:
+                      type: string
+                    type: array
+                type: object
+              ee_dns_policy:
+                type: string
+              ee_host_network:
+                type: boolean
+              web_dns_config:
+                properties:
+                  nameservers:
+                    items:
+                      type: string
+                    type: array
+                  options:
+                    items:
+                      properties:
+                        name:
+                          type: string
+                        value:
+                          type: string
+                      type: object
+                    type: array
+                  searches:
+                    items:
+                      type: string
+                    type: array
+                type: object
+              web_dns_policy:
+                type: string
+              web_host_network:
+                type: boolean
+              task_dns_config:
+                properties:
+                  nameservers:
+                    items:
+                      type: string
+                    type: array
+                  options:
+                    items:
+                      properties:
+                        name:
+                          type: string
+                        value:
+                          type: string
+                      type: object
+                    type: array
+                  searches:
+                    items:
+                      type: string
+                    type: array
+                type: object
+              task_dns_policy:
+                type: string
+              task_host_network:
+                type: boolean
+              init_dns_config:
```

Comment:
@rakesh561 Is there a use case for having this for the init container?  I don't think we should add it if there isn't a scenario that it would be used.